engine/src/test/java/com/datatorrent/stram/plan/logical/LogicalPlanTest.java

```diff
@@ -1273,4 +1276,22 @@ public void testValidInputOperatorDeclaration()
     dag.validate();
   }
   */
+
+  @Test
+  public void testVisitor()
+  {
+    LogicalPlan dag = new LogicalPlan();
+    InputNodeTest.TestInputOperator input = dag.addOperator("Input", new InputNodeTest.TestInputOperator());
+    GenericTestOperator op = dag.addOperator("Calc", new GenericTestOperator());
+    dag.addStream("s1", input.output, op.inport1);
+
+    Configuration conf = new Configuration();
+    conf.set("apex.dagSetupPlugins", "com.datatorrent.stram.plan.logical.PropertyInjectorVisitor");
+    conf.set("propertyVisitor.Path","/visitortests.properties");
+    dag.setupPlugins(conf);
```

Comment:
Let's have the framework classes do this work and test that instead of calling the validations method directly